docs/src/code-examples-content.js

```diff
@@ -1,593 +0,0 @@
-
```

Comment:
This is in the `.gitignore` and should not be included. Was removed before, but seems to have snuck back in.

Reply:
yeah I think anyone who has it locally from before the change to `.gitignore` might end up checking it back in, unfortunately